Settings.editme

```diff
@@ -75,14 +75,6 @@
    #                                 #### CUSTOM AI CONFIGURATION  ####
    # NOTE: All the stuff here is mainly here to facilitate testing. But feel free to play around with the settings here!
    
-   # Define the number of calculation steps the AI should perform per time frame to compute firing solutions.
```

Comment:
This setting is no longer required, as we always have the perfect accuracy with just a single solved equation.